lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
@rdubya so I assumed we used PG because activerecord methods are scoped using some methods from PG.  If so then I would rather leverage the difficulties of that then clone all of the methods which call PG ones in AR.  

I agree the correct long term solution may be to abstract/remove PG in a way which we do not need to stomp on the same constant and "pretend".